util/stop_watch.h

```diff
@@ -133,4 +133,35 @@ class StopWatchNano {
   uint64_t start_;
 };
 
+class StopWatchCPUMicros {
```

Comment:
why not adapt StopWatchNano class to optionally provide ElapsedCPUMicros()? These two classes are very similar.

Reply:
oh i see ... bad naming on the StopWatchNano. Can you use nanos too so you can just augment the same class?

Reply:
It doesn't seem worth having essentially a copy of the same class to save dividing by 1000 for those who want microseconds (and thus won't be as sensitive to an extra divide instruction)

You could also have this one inherit from StopWatchNano and provide the ElapsedMicros() function.